client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -63,123 +68,136 @@ class TransferDomainPrecheck extends React.PureComponent {
 		} );
 	};
 
-	getSection( heading, message, explanation, position, button ) {
-		const header = (
-			<div className="transfer-domain-step__section">
-				<span className="transfer-domain-step__section-heading-number">{ position }</span>
-				<div>
-					<strong className="transfer-domain-step__section-heading">{ heading }</strong>
-					<div className="transfer-domain-step__section-message">{ message }</div>
-				</div>
-			</div>
-		);
+	showNextStep = () => {
+		this.setState( { currentStep: this.state.currentStep + 1 } );
+	};
 
-		const cardClasses = classNames( {
-			'hide-secondary': ! button,
+	getSection( heading, message, buttonText, position, lockStatus ) {
+		const { currentStep, loading, unlocked } = this.state;
+		const sectionClasses = classNames( 'transfer-domain-step__section', {
+			'is-expanded': position === currentStep,
+			'is-complete': position < currentStep,
 		} );
 
+		const sectionIcon =
+			currentStep > position ? <Gridicon icon="checkmark-circle" size={ 36 } /> : position;
+
 		return (
-			<FoldableCard
-				className={ cardClasses }
-				header={ header }
-				summary={ button }
-				expandedSummary={ button }
-			>
-				<div className="transfer-domain-step__section-explanation">{ explanation }</div>
-			</FoldableCard>
+			<Card compact>
+				<div className={ sectionClasses }>
+					<span className="transfer-domain-step__section-heading-number">{ sectionIcon }</span>
+					<div className="transfer-domain-step__section-text">
+						<div className="transfer-domain-step__section-heading">
+							<strong>{ heading }</strong>
+							{ lockStatus }
+						</div>
+						{ position === currentStep && (
+							<div>
+								<div className="transfer-domain-step__section-message">{ message }</div>
+								<Button
+									compact
+									onClick={ unlocked ? this.showNextStep : this.refreshStatus }
+									disabled={ loading }
+								>
+									{ buttonText }
+								</Button>
+							</div>
+						) }
+					</div>
+				</div>
+			</Card>
 		);
 	}
 
 	getStatusMessage() {
 		const { translate } = this.props;
-		const { unlocked } = this.state;
+		const { unlocked, loading } = this.state;
 
 		const heading = unlocked
 			? translate( 'Domain is unlocked.' )
 			: translate( 'Unlock the domain.' );
 		const message = unlocked
 			? translate( 'Your domain is unlocked at your current registrar.' )
 			: translate(
-					"You'll need to unlock the domain at your current registrar before we can move it."
+					"Your domain is locked to prevent unauthorized transfers. You'll need to unlock " +
+						'it before we can move it.'
 				);
-		const explanation = translate(
-			'Your current domain provider has locked the domain to prevent unauthorized transfers.'
-		);
+		const buttonText = loading ? translate( 'Checking…' ) : translate( "I've unlocked my domain" );
```

Comment:
Hi! I've found a possible matching string that has already been translated **62** times:
`translate( 'Checking...' )` *ES Score: 21*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/domain-transfer-precheck-i2)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).